biogrid/libs/biogrid-simulator/src/lib/community/ruralarea.spec.ts

```diff
@@ -1,30 +1,49 @@
 import { Building } from '../building';
 import { RuralArea } from './';
 
-
 describe('tests for RuralArea class', () => {
+  const x = 10,
+    y = 10;
   test('getBuildingById()', () => {
-    const buildings : Building[] = [new Building(5), new Building(10), new Building(20)];
-    const ruralarea = new RuralArea(buildings);
+    const buildings: Building[] = [
+      new Building(5, 2, 3),
```

Comment:
can you do the in-line comment here as well? 
`new Building( /* param1= */ 5, /* param2= */ 2,  /* param3= */ 3)`
Should be fine to just do it on the first building in the list